types/deep-equal-in-any-order/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for deep-equal-in-any-order 1.0
+// Project: https://github.com/oprogramador/deep-equal-in-any-order#readme
+// Definitions by: Bastien Caudan <https://github.com/bcaudan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```

Comment:
It doesn't seem necessary to reference `chair` here in the typings itself, since no types from `chai` are actually used. 

I'd think the test code / user code should be the one referencing the chai typings.

Reply:
removed in 9326347